### ISSUE TITLE
Add badges to the 3rd category in the Achievement Room

### ISSUE THREAD

**Issue by yatna** [systers/malaria-app-android#120]
Currently in the Achievement Room, only the first 2 categories display badges based on their associated score. This has to be extended for the third category as well . The third category should be based on the point accumulated by planning trips using the app's trip planner facility. Any ideas on how to distribute points and which badges to keep are welcome, but the basic philosophy behind any idea should be to promote the users to use the app's trip planner.

**Comment by yatna** [comment]
This issue is free for anyone to take up. @chhavip can you please label it as free. Thank You.

**Comment by anubhakushwaha** [comment]
@yatna This is a published task in GCI-2016.

**Comment by anubhakushwaha** [comment]
@yatna This task was well executed by one of the GCI student,here is the work done by the student.
[5170912315834368_1482271476_malaria app.zip](https://github.com/systers/malaria-app-android/files/712914/5170912315834368_1482271476_malaria.app.zip) It was well executed with the reasons for each badge as well as mentioned in theses slides https://docs.google.com/presentation/d/1iKyyktVKNBQojr6cjh3Ze_RFvYbIyo-EGWvDky_ySDw/edit?usp=sharing .I am removing the tag so that more work can be done by other contributors so that at the end we choose the best work.

**Comment by Shreya001** [comment]
is this issue resolved?

**Comment by anubhakushwaha** [comment]
@Shreya001  You can go ahead with this, let us know incase you have any query.

**Comment by Shreya001** [comment]
We have to add badges for the app's trip planner and promote using it, So
we can add badges for planned the first trip, early bird (if the person
sets the departure time of trip early in morning), badges based on the
number of trips planned. We can also add the description of each badge as
currently it just displays the name. Let me know if liked the idea, I would
then share the badges with you and send pr for the same.

Thanks
Shreya

**Comment by Shreya001** [comment]
@anubhakushwaha We have to add badges for the app's trip planner and promote using it, So we can add badges for planned the first trip, early bird (if the person sets the departure time of trip early in morning), badges based on the number of trips planned. We can also add the description of each badge as currently it just displays the name. Let me know if liked the idea, I would then share the badges with you and send pr for the same

**Comment by anubhakushwaha** [comment]
@Shreya001  Yeah I like your ideas, can you draft them somewhere(a google doc maybe)? and share with us so that we can add/edit there and then you can implement the logic for the same :1
